src/radiation/radiation_system.hpp

```diff
@@ -1678,6 +1695,10 @@ void RadSystem<problem_t>::AddSourceTerms(array_t &consVar, arrayconst_t &radEne
 				} // END NEWTON-RAPHSON LOOP
 
 				AMREX_ALWAYS_ASSERT_WITH_MESSAGE(n < maxIter, "Newton-Raphson iteration failed to converge!");
```

Comment:
Is there a reason to leave the old abort statements in the code?

Reply:
I kept it there just to let abort earlier on CPU. This is particularly useful for development because you know exactly when and where it aborts.

Reply:
Can you change it to `AMREX_ASSERT_WITH_MESSAGE`? Then it only has different behavior when `-DCMAKE_BUILD_TYPE=Debug`, rather than CPU vs GPU (which I think is confusing).

Reply:
OK.